### PR TITLE
fix yeoman-ui tab close logic

### PR DESCRIPTION
**Pull request by slavik-lvovsky**

### CODE REVIEW

**Comment by tomer-epstein** [comment]
image

1st time works fine.
2nd time show install notification twice, (install depends didn't happen)

File: backend/src/yeomanui.ts [hunk 1]
```diff
@@ -152,24 +152,17 @@ export class YeomanUI {
...
-        if (err) {
-          message = `${generatorName} generator failed.\n\n${this.getErrorInfo(err)}`;
-          this.logError(err, message);
-          this.youiEvents.doGeneratorDone(false, message, destinationRoot);
-        } else {
-          message = `The '${generatorName}' project has been generated.`;
-          this.logger.debug("done running yeomanui! " + message + ` You can find it at ${destinationRoot}`);
-          this.youiEvents.doGeneratorDone(true, message, destinationRoot);
-        }
+      const destinationRoot = this.gen.destinationRoot();
```
Comment (tomer-epstein) [inline backend/src/yeomanui.ts]: Must get the destRoot after generator has been run.
The destRoot might be changed during run.

Comment (slavik-lvovsky) [reply backend/src/yeomanui.ts]: Done

**Comment by slavik-lvovsky** [comment]
Done

**Pull request merged**